### ISSUE TITLE
Hard to fix problem: FIles not saving

### ISSUE THREAD

**Issue by hg0428** [thebaselab/codeapp#456]
The file says its saved, it runs fine, and I press command s, but when I exit and reopen the app I, nothing is saved.

**Comment by bummoblizard** [comment]
A screen recording would be great.

**Comment by hg0428** [comment]
> A screen recording would be great. 

Sorry I'm late responding.
The reason I can't get a screen recording is because I have not yet found the cause, and so I can not predict when it is going to happen.

I just recently got better connection, so wifi is no longer a problem for me.

**Comment by hg0428** [comment]
I have better connection now, but I am still unable to determine the cause of the problem, and I can't just record the screen 24/7 waiting for something to happen

**Comment by hg0428** [comment]
I did one, but my wifi isn't good enough right now; I'm on vacation. When I get back I will upload it.

**Comment by dideldudeldei** [comment]
Yup, facing the same from time to time. Would be great if there would be a „Saving… do not close the app“ kind of pop-up
